[0m08:18:52.680801 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7faba9156130>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fabaa9f7550>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fabaa9f7b50>]}


============================== 08:18:52.685791 | 3e7c0172-39c8-4f5e-ae40-b490a9ff73b3 ==============================
[0m08:18:52.685791 [info ] [MainThread]: Running with dbt=1.8.6
[0m08:18:52.686312 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'version_check': 'True', 'profiles_dir': '/Users/dellacorte/py-projects/data-engineering/crm-dbt-datawarehouse-etl', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'log_format': 'default', 'static_parser': 'True', 'target_path': 'None', 'invocation_command': 'dbt init', 'send_anonymous_usage_stats': 'True'}
[0m08:20:10.491704 [debug] [MainThread]: Starter project path: /Users/dellacorte/opt/anaconda3/lib/python3.9/site-packages/dbt/include/starter_project
[0m08:20:10.510386 [info ] [MainThread]: 
Your new dbt project "sales_dbt" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m08:20:10.511741 [info ] [MainThread]: Setting up your profile.
[0m08:21:00.003878 [info ] [MainThread]: Profile sales_dbt written to /Users/dellacorte/py-projects/data-engineering/crm-dbt-datawarehouse-etl/profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m08:21:00.024258 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 127.423935, "process_user_time": 0.88101, "process_kernel_time": 0.485475, "process_mem_max_rss": "87748608", "process_in_blocks": "0", "process_out_blocks": "0"}
[0m08:21:00.026338 [debug] [MainThread]: Command `dbt init` succeeded at 08:21:00.025576 after 127.43 seconds
[0m08:21:00.027452 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7faba9156130>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fabaab4d2e0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fabaa9baeb0>]}
[0m08:21:00.028401 [debug] [MainThread]: Flushing usage events
[0m10:32:59.088123 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f8701477160>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f8702dbe6d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f8702dbeb80>]}


============================== 10:32:59.093926 | 1ee61064-2b0d-4fb0-bafe-b801ef5a0f1b ==============================
[0m10:32:59.093926 [info ] [MainThread]: Running with dbt=1.8.6
[0m10:32:59.094441 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/dellacorte/.dbt', 'debug': 'False', 'warn_error': 'None', 'log_path': 'logs', 'fail_fast': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'invocation_command': 'dbt debug', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m10:32:59.104213 [info ] [MainThread]: dbt version: 1.8.6
[0m10:32:59.104485 [info ] [MainThread]: python version: 3.9.13
[0m10:32:59.104715 [info ] [MainThread]: python path: /Users/dellacorte/opt/anaconda3/bin/python
[0m10:32:59.104949 [info ] [MainThread]: os info: macOS-10.16-x86_64-i386-64bit
[0m10:32:59.105179 [info ] [MainThread]: Using profiles dir at /Users/dellacorte/.dbt
[0m10:32:59.105390 [info ] [MainThread]: Using profiles.yml file at /Users/dellacorte/.dbt/profiles.yml
[0m10:32:59.105600 [info ] [MainThread]: Using dbt_project.yml file at /Users/dellacorte/py-projects/data-engineering/crm-dbt-datawarehouse-etl/dbt_project.yml
[0m10:32:59.105824 [info ] [MainThread]: Configuration:
[0m10:32:59.106025 [info ] [MainThread]:   profiles.yml file [[31mERROR not found[0m]
[0m10:32:59.106229 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m10:32:59.106433 [info ] [MainThread]: Required dependencies:
[0m10:32:59.106702 [debug] [MainThread]: Executing "git --help"
[0m10:32:59.174635 [debug] [MainThread]: STDOUT: "b"usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]\n           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]\n           [--super-prefix=<path>] [--config-env=<name>=<envvar>]\n           <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone     Clone a repository into a new directory\n   init      Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add       Add file contents to the index\n   mv        Move or rename a file, a directory, or a symlink\n   restore   Restore working tree files\n   rm        Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect    Use binary search to find the commit that introduced a bug\n   diff      Show changes between commits, commit and working tree, etc\n   grep      Print lines matching a pattern\n   log       Show commit logs\n   show      Show various types of objects\n   status    Show the working tree status\n\ngrow, mark and tweak your common history\n   branch    List, create, or delete branches\n   commit    Record changes to the repository\n   merge     Join two or more development histories together\n   rebase    Reapply commits on top of another base tip\n   reset     Reset current HEAD to the specified state\n   switch    Switch branches\n   tag       Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch     Download objects and refs from another repository\n   pull      Fetch from and integrate with another repository or a local branch\n   push      Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m10:32:59.175583 [debug] [MainThread]: STDERR: "b''"
[0m10:32:59.175903 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m10:32:59.176194 [info ] [MainThread]: Connection test skipped since no profile was found
[0m10:32:59.176535 [info ] [MainThread]: [31m2 checks failed:[0m
[0m10:32:59.176773 [info ] [MainThread]: dbt looked for a profiles.yml file in /Users/dellacorte/.dbt/profiles.yml, but did
not find one. For more information on configuring your profile, consult the
documentation:

https://docs.getdbt.com/docs/configure-your-profile


[0m10:32:59.177021 [info ] [MainThread]: Project loading failed for the following reason:
 project path </Users/dellacorte/py-projects/data-engineering/crm-dbt-datawarehouse-etl/dbt_project.yml> not found

[0m10:32:59.181330 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_wall_clock_time": 0.17270416, "process_user_time": 0.841161, "process_kernel_time": 0.390371, "process_mem_max_rss": "86642688", "command_success": false, "process_in_blocks": "0", "process_out_blocks": "0"}
[0m10:32:59.182310 [debug] [MainThread]: Command `dbt debug` failed at 10:32:59.181901 after 0.17 seconds
[0m10:32:59.182861 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f8701477160>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f86f0092f70>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f86f00929a0>]}
[0m10:32:59.183281 [debug] [MainThread]: Flushing usage events
